lib/mayaUsd/fileio/jobs/jobArgs.cpp

```diff
@@ -75,159 +76,8 @@ TF_DEFINE_PRIVATE_TOKENS(
 // clang-format on
 
 namespace {
-/// Extracts a bool at \p key from \p userArgs, or false if it can't extract.
```

Comment:
All changes are merely moving all these function n a public namespace for reuse and chaning the call sites to use the new names.